azure-pipelines/main-pipeline.yml

```diff
@@ -62,6 +63,29 @@ stages:
     
     steps:
 
+    
+    # Static Analysis with SonarCloud. Pointing to Java Binaries and exluding the JDK
+    # Tar file that will be installed in the pipeline. This step scans all services.
+
+    - task: SonarCloudPrepare@1
+      inputs:
+        SonarCloud: 'SonarCloud'
+        organization: '$(sonarOrg)'
+        scannerMode: 'CLI'
+        configMode: 'manual'
+        cliProjectKey: '$(sonarProjectKey)'
+        cliProjectName: '$(sonarProjectName)'
+        cliSources: '.'
+        extraProperties: |
+            sonar.java.binaries=**/src
```

Comment:
if you're sure this is a good setting for sonar.java.binaries, then ok

Reply:
Fixed with sonarcloud rework.